src/Diagnostics.DataProviders/DataProviders/MdmDataProvider.cs

```diff
@@ -266,6 +271,12 @@ public async Task<IEnumerable<DataTable>> GetMultipleTimeSeriesAsync(DateTime st
                         queryParameters.Add(mdmParameter);
                     }
                 }
+
+                if (dashboard == "WAWS Shoebox/Web Apps/Per Resource")
+                {
+                    AddRemainingParametersForWebApps(queryParameters, resourceId, seriesResolutionInMinutes);
+                }
```

Comment:
Now that we have this GenericMdm logic, I can see multiple partners who want to also benefit by adding additional information to the metadata. If we continue with this style of logic then I can see this list growing in the future. Not that its an urgent problem but we should probably think of a dynamic way of doing this.

Reply:
Agreed. For GenericMdm logic, we need to come up with a better way of handling this. Currently this change is not impacting the GenericMdm provider at all.